articles/_a-propos/que-publier-et-comment-le-publier.md

```diff
@@ -0,0 +1,131 @@
+* les collectivités territoriales (villes, conseils généraux, conseils régionaux) ;
+* les entreprises qui assurent une mission de service public (SNCF, RATP, Caisse des dépôts).
+
+## Que publier
+
+Vaste sujet. Pour commencer, nous vous conseillons de faire la distinction entre les jeux de données et les agrégats de documents isolés. Quelles différences entre un jeu de données et un document isolé ? Les jeux de données sont suffisamment volumineux et structurés pour être analysés par une machine, ce qui n’est pas toujours le cas des documents isolés. Dit autrement, mettre des informations sur internet ne suffit pas, hélas, encore faut-il que les informations mises en ligne soient ordonnées et lisibles par une machine — sans quoi elles ne pourront être utilisées.
+
+Par exemple, le taux de réussite à l’examen du baccalauréat ne constitue pas un jeu de données. À l’échelle nationale, il s’agit d’un simple chiffre, un document isolé. En revanche, si un tableau Excel (ou un fichier CSV) contient le taux de réussite au baccalauréat de _tous_ les lycées français, on peut dans ces conditions commencer à parler de jeu de données, car ces données peuvent être croisées, recoupées avec d’autres données, dans le but de produire des analyses.
+
+Un bon test pour vos données consiste donc à évaluer leur modularité : vos données peuvent-elles facilement être combinées avec d’autres jeux de données ? Par exemple, les données sur les dépenses de santé d’une région peuvent être combinées avec les données démographiques de cette même région dans le but d’évaluer l’efficacité des dépenses de santé sur ce territoire.
+
+Si vous ne savez pas par où commencer, nous vous recommandons de faire simple, en commençant par des jeux de données faciles à rassembler et susceptibles d’intéresser le grand public.
+
+Voici quelques exemples tout sauf exhaustifs :
+
+* les mesures de qualité de l’air ;
+* les adresses des services publics ;
+* les statistiques du tourisme ;
+* les résultats électoraux ;
+* les horaires des transports publics ;
+* les impôts et et le budget.
+
+## Comment le publier
+
+### Choisir un format de publication standardisé
+
+Les données publiées sur data.gouv.fr ont vocation à être consultées et réutilisées. Pour que vos données puissent être réutilisées par des tiers, nous vous conseillons de les mettre en forme selon des standards ouverts et communément adoptés dans le secteur au sein duquel vous évoluez.
+
+#### Données tabulaires
+
+Les données tabulaires (feuilles de calcul), gagnent à être publiées sous la forme de fichiers CSV (_comma-separated values_), reconnaissables à leur extension en `.csv`. Si le fichier que vous avez entre les mains est un fichier Excel, nous vous conseillons d’exporter vos données au format CSV. [Voir comment exporter des données au format CSV à partir d’Excel](https://support.office.com/fr-fr/article/importer-ou-exporter-des-fichiers-texte-txt-ou-csv-5250ac4c-663c-47ce-937b-339e391393ba)
```

Comment:
En regardant rapidement (https://stackoverflow.com/questions/4221176/excel-to-csv-with-utf8-encoding) cette méthode ne produit pas de l'UTF-8, ce qui est fortement conseillé... Peut-être faut-il pointer vers une méthode alternative ?

Reply:
Oui, j’en ai parlé avec @cquest. J’ai changé le lien présent dans l’article. L’article redirige maintenant vers [un article d’IBM](https://www.ibm.com/support/knowledgecenter/fr/SSWU4L/WebLanding/imc_WebLanding/WebLanding_q_a_watson_assistant/Saving_a_CSV_file_with_UTF-8_encoding.html).